ansible/playbook.yml

```diff
@@ -0,0 +1,163 @@
+- become: true
+  hosts: all
+  name: Impostor-install
+  gather_facts: true
+  vars:
+    impostor_nightly_zip_path: "/home/ec2-user/Impostor-Server-latest.zip"
+    impostor_path: "/home/ec2-user/Impostor"
+    impostor_plugins: "/home/ec2-user/Impostor/Plugins"
+    impostor_user: "ec2-user"
+    public_port: "443"
+    private_port: "22023"
+    http_port: "22022"
+    
+  tasks:
+
+  - name: "Impostor on RHEL-based systems"
+    when: ansible_os_family == "RedHat"
+    block:
+
+      - name: "Make sure system is up-to-date"
+        vars:
+          ansible_python_interpreter: /usr/bin/python
+        ansible.builtin.yum:
+          name: '*'
+          state: latest
+          update_cache: true
+
+      - name: Get our external IP address to set in config.json
```

Comment:
We should no need to pull the external config and supply a domain name. In my PR I am going to be passing in domain values to allow ansible to set the variable. It will be the game_domain.
https://github.com/rayprogramming/terraform-aws-impostor-server/pull/11/files#diff-6c27bcefa33af3496292d0448a95189bc1f75e38a9a6c9970e4490bef4c98a84R45

Reply:
Yeah I figured we'd need to pull in the LB IP info with TF but I wasn't sure, this was more for running it in my own homelab but I can remove that extra stuff if you don't need it.